images.yaml

```diff
@@ -583,6 +583,24 @@
 - name: quay.io/calico/kube-controllers
   patterns:
   - pattern: '>= v3.9.5'
+- name: quay.io/calico/kube-controllers
+  overrideRepoName: calico-crd-installer
+  patterns:
+  - pattern: '>= v3.16.0'
+    customImages:
+    - dockerfileOptions:
+      - |
+        FROM quay.io/giantswarm/crd-installer:0.0.0-add8f5797322a6e44489740b17a59c0a64ae36e7 AS installer
+        FROM quay.io/giantswarm/alpine:3.13.5 AS downloader
+        WORKDIR /tmp/crd-installer
+        COPY --from=0 /usr/bin/kube-controllers /crd-installer/kube-controllers
+        COPY --from=installer /scripts /scripts
+        RUN CALICO_VERSION=$(/crd-installer/kube-controllers -version) && \
+            /scripts/download-calico-crds.sh $CALICO_VERSION
```

Comment:
Okay scratch my question in the other review. Should have checked there here first.